ExamplesForHomalg/PackageInfo.g

```diff
@@ -103,7 +103,7 @@ AvailabilityTest := function()
     return true;
   end,
 
-
+TestFile := "maketest.g",
```

Comment:
Does this file exist?

Reply:
Yes, it is in the repository.

Reply:
Ah, you probably mean `4ti2Interface/PackageInfo.g`. Correct, fixed.

Reply:
Did this resolve your question?

Reply:
yes